.env

```diff
@@ -0,0 +1,3 @@
+PORT=4000
+SECRET='BANGARANG PETER!!...LOST BOYS, LOST BOYS.. WAKE UP!'
```

Comment:
Its not a secret no more!

Reply:
make sure to add a .gitignore file and add the env file there so it doesn't get pushed to github :)